package cms_8120;
// @设备类型定义

enum DEV_TYPE {
    
    DEV_TYPE_DVR                = 50000 ; // DVR
    DEV_TYPE_ALARM_HOST         = 50001 ; //报警主机
    DEV_TYPE_GATE               = 50002 ; //门禁
    DEV_TYPE_TALK               = 50003 ; //对讲
    DEV_TYPE_DECODER            = 50004 ; //解码器
    DEV_TYPE_VIP                = 50005 ; //视频综合平台
    DEV_TYPE_SC                 = 50006 ; //防护舱(Safety Cabin)
    DEV_TYPE_IPC                = 50007 ; // IPC
    DEV_TYPE_DVS                = 50008 ; // DVS
    DEV_TYPE_PCDVR              = 50009 ; // PCDVR
    DEV_TYPE_AH_NETWORK_MODULE  = 50010 ; //报警主机网络模块
    DEV_TYPE_CVR                = 50011 ; // CVR
    DEV_TYPE_NVR                = 50012 ; // NVR
    DEV_TYPE_ENVIRONMENT        = 50013 ; // 环境量
    DEV_TYPE_FIRE               = 50014 ; // 消防
    DEV_TYPE_CLOUD_STORAGE      = 50015 ; // 云存储
    DEV_TYPE_GUN                = 50016 ; // 枪弹柜(昆明红河邮政)
    DEV_TYPE_YS7		= 50017	; //萤石云设备
    DEV_TYPE_KMS                = 50050 ; // KMS
    DEV_TYPE_OBJECTCLOUD        = 50060 ; // 标准云-对象存储
}
